src/Private/Helpers.ps1

```diff
@@ -1415,7 +1442,7 @@ function New-PodeRequestException {
 
 function ConvertTo-PodeResponseContent {
```

Comment:
There's quite a few of these internal functions where they have parameters flagged with `ValueFromPipeline = $true`, but I never actually use the pipeline - mostly to help with performance, and directly pass the object instead.

`ConvertTo-PodeResponseContent` and `ConvertFrom-PodeHeaderQValue` are two such examples, but there are others above and below which I've crossed.

I'm thinking we just remove the attribute instead, from parameters on functions where the values just aren't being passed via pipeline.

Reply:
are internal function we can do it